.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
-**Commit Message**
+**Description**
 
 <!--
-Please write the commit message here, which
-typically talks about the description / what this commit does / why we need it
+Please write the description here, which typically talks about the description / what this PR does / why we need it.
+After the PR is merged, the message here will be included in the commit message after the PR title (which is in the format of "<tag>: add foo for bar").
```

Comment:
@Xunzhuo hope this helps you understand what i meant ...

Reply:
description is more accurate, i thought we need to to paste the commit message in PR description